transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/query/Constraint.groovy

```diff
@@ -311,22 +318,30 @@ class ConceptConstraint extends Constraint {
 
 @Canonical
 class StudyNameConstraint extends Constraint {
+    static String constraintName = "study_name"
+
     String studyId
 }
 
 @Canonical
 class StudyObjectConstraint extends Constraint {
+    static String constraintName = "study"
+
     Study study
 }
 
 @Canonical
 class NullConstraint extends Constraint {
+    static String constraintName = "null"
+
     @BindUsing({ obj, source -> ConstraintFactory.bindField(obj, 'field', source['field']) })
     Field field
 }
 
 @Canonical
 class RowValueConstraint extends Constraint {
+    static String getName() { throw new UnsupportedOperationException("internal use") }
```

Comment:
We don't for this class, this is just to document why you shouldn't be using this. Not having a static name property would give you a property not found exception and doesn't tell you why.